iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/ConnectableConnection.java

```diff
@@ -77,6 +77,20 @@ public void apply(Network network, NamingStrategy namingStrategy, boolean throwE
         }
     }
 
+    @Override
+    public NetworkModificationImpact hasImpactOnNetwork(Network network) {
```

Comment:
I still don't understand your check (and for disconnection too). For me, the `side` attribute must be used in this method.

Reply:
After a talk with @pet-mit, side must be taken into account, even if there is no impact on powsybl-open-rao.